script/lib/utils.js

```diff
@@ -19,7 +24,22 @@ function getAbsoluteElectronExec () {
   return path.resolve(__dirname, '../../..', getElectronExec())
 }
 
+async function getCurrentBranch (gitDir) {
+  const gitArgs = ['rev-parse', '--abbrev-ref', 'HEAD']
+  const branchDetails = await GitProcess.exec(gitArgs, gitDir)
+  if (branchDetails.exitCode === 0) {
+    const currentBranch = branchDetails.stdout.trim()
+    console.log(`${pass} current git branch is: ${currentBranch}`)
+    return currentBranch
+  } else {
+    const error = GitProcess.parseError(branchDetails.stderr)
+    console.log(`${fail} couldn't get details current branch: `, error)
+    process.exit(1)
```

Comment:
I wonder if this should throw instead of just exiting? 🤔 That said, everywhere it's used exiting is the right behavior. Your call